src/Apps/Settings/Routes/MyCollection/Components/HowMyCollectionWorks.tsx

```diff
@@ -0,0 +1,51 @@
+import {
+  StepsWithImage,
+  StepsWithImageDataType,
+} from "Components/StepsWithImage"
+import { resized } from "Utils/resized"
+
+const imageHeight = 320
+const imageWidth = 450
+const resizeParams = { width: imageWidth, height: imageHeight, quality: 100 }
```

Comment:
#superminor - could just inline the height and width vars in this line, versus defining them separately